### ISSUE TITLE
Freepbx 16

### ISSUE THREAD

**Issue by slv007** [POSSA/freepbx-trunk-balancing#45]
This module works ok if i upgrade to FreePBX 16? anyone tested?

**Comment by sickfuck666** [comment]
Yep, but you need to use fixed version - https://github.com/pbx-profi/freepbx-trunk-balancing
Here is ready to install file - https://cloud.mail.ru/public/Ms3U/hJuzaL6E2
